honeybee_radiance/properties.py

```diff
@@ -32,19 +31,21 @@ def modifier(self):
 
     @modifier.setter
     def modifier(self, value):
-        if value:
-            assert isinstance(value, Material), \
-                'Expected Radiance modifier not {}'.format(type(value))
-        self._modifier = value
+        if hasattr(value, 'can_be_modifier') and value.can_be_modifier:
+                self._modifier = value
+        else:
+            raise TypeError(
+                '{} is not a valid Radiance modifier.'.format(type(value))
+            )
 
     @property
     def is_modifier_set_by_user(self):
-        """Check if construction is set by user."""
+        """Check if modifier is set by user."""
         return self._modifier is not None
 
     @property
     def is_modifier_set_by_zone(self):
```

Comment:
Interesting! Does this mean you thinking of making a Radiance MaterialSet that can be applied on the zone level (similar to EnergyPlus ConstructionSet)?

Reply:
If so, I just wanted to say that I fully support the idea.

Reply:
Yes. Now that we are going in the direction of construction sets I felt it is probably a good idea to sync between our sets between radiance and energy model. Great that you are also supporting the idea.